src/qibolab/platforms/dummy.py

```diff
@@ -38,12 +39,15 @@ def disconnect(self):
     def to_sequence(self, sequence, gate):  # pragma: no cover
         raise_error(NotImplementedError)
 
-    def execute_pulse_sequence(self, sequence, nshots=None):  # pragma: no cover
-        time.sleep(self.settings.get("sleep_time"))
+    def execute_pulse_sequence(self, sequence, nshots=None, wait_time=None):
+        if wait_time is None:
+            wait_time = self.settings.get("sleep_time")
 
         if nshots is None:
             nshots = self.settings["settings"]["hardware_avg"]
 
+        time.sleep(wait_time)
```

Comment:
Why do we need a `wait_time` at all?

Reply:
The `wait_time` which I introduced in #259, with maybe not the best name, is the time we wait before each shot. This is needed when using real qubits because we need to wait for all qubits to relax to their ground state after every shot, so that in the next shot we start from state |00...0> (otherwise in Monte Carlo terms the shots will be correlated which we don't want). In the dummy platform is not very relevant, I think we only did it to test qibocal's qq-live, otherwise execution is really fast and we get all points together.

Originally this time was a parameter in the runcard, called `repetition_duration` for qblox. I asked @andrea-pasquale to make it a function argument because this way it is easier to control. In some calibrations such as the resonator and qubit spectroscopy we actually don't need to wait for the qubits to relax so we can even set it to 0. Normally this is on the order of 100us (assuming T1 1-10us) but when you make 1000 shots and a scan of >100 points it becomes non-negligible.

Reply:
I see, thank you, @stavros11, for the thorough explanations.

I'm also not extremely good at naming things. In this case, I believe that in MC you usually call it *correlation length*, while in physical system this is closer to a relaxation time.

No problem at all with the concept, and it is definitely needed. The only suggestion I have for @andrea-pasquale  is to take the comment above and fit in a docstring: it is already a great explanation on its own xD

Reply:
Done.